guides/v2.1/coding-standards/docblock-standard-javascript.md

```diff
@@ -139,7 +141,7 @@ project.MyClass.EmptyNodeList&nbsp;=&nbsp;function()&nbsp;{
 </pre>
 
 
-<h4 id="frontend-dev-guide-javascript-js-coding-docblock-interface">@interface</h4>
+#### @interface
```

Comment:
Please append `{#frontend-dev-guide-javascript-js-coding-docblock-interface}` to preserve the custom id.